Update data-collection-windows-events.md (#40)
MicrosoftDocs/azure-monitor-docs: articles/azure-monitor/agents/data-collection-windows-events.md

[!INCLUDE [azure-monitor-cost-optimization](../../../includes/azure-monitor-cost-optimization.md)]

> > [!NOTE]
> > AMA uses the system API [EvtSubscribe](/windows/win32/api/winevt/nf-winevt-evtsubscribe) to subscribe to Windows Event Logs. Windows OS does not allow subscribing to Windows Event Logs of type Analytic/Debug channels. You cannot subscribe to an Analytic or Debug channel then you cannot collect or export data from those channels to be ingested in Log Analytics.

Reviewer: ```suggestion
> [!NOTE]
> AMA uses the [EvtSubscribe](/windows/win32/api/winevt/nf-winevt-evtsubscribe) system API to subscribe to Windows Event Logs. Windows OS does not allow subscribing to Windows Event Logs of type Analytic/Debug channels. Therefore, you cannot collect or export data from Analytic and Debug channels to a Log Analytics workspace.
```
